radar-upload-frontend/vue.config.js

```diff
@@ -1,7 +1,9 @@
 // vue.config.js
 module.exports = {
 
-  publicPath: '/uploader/',
+  publicPath: process.env.NODE_ENV === 'development'
+    ? '/'
+    : '/uploader',
```

Comment:
Not sure about this, shouldn't this logic be solved with the `.env.production` and `.env.development` files?

Reply:
I did not touch the cookies yet. 
My local (development) environment variables do not contain '/uploader'. It's originally '/', but @nivemaham  seems want to have the '/uploader' in the build so I changed that to not affect my development.

Reply:
My intention was to add the `uploader` as the baseurl for both `development` and `production` modes. Thus, the app would run on `http://localhost:8080/uploader` instead of `http://localhost:8080`. The reason was to have a locally deploy-able set of containers that are already configured to work together. I do think it is a problem to run the development version with the `uploader` baseUrl ( given that it is documented) . If not, you can add an env varibale on the env.file.

Reply:
I don't mind if it the `localhost:8080/uploader` or `localhost:8080`. Just want the redirect url work on my development. If like to change to /uploader in development mode, please change the authentication redirect url as well.